content/en/tracing/runtime_metrics/go.md

```diff
@@ -27,28 +27,29 @@ tracer.Start(tracer.WithRuntimeMetrics())
 
 View runtime metrics in correlation with your Go services on the [Service page][1] in Datadog.
 
-By default, runtime metrics from your application are sent every 10 seconds to the Datadog Agent with DogStatsD over port `8125`. Make sure that [DogStatsD is enabled for the Agent][2]. If your Datadog Agent DogStatsD address differs from the default `localhost:8125`, use the `WithDogstatsdAddress` option or the environment variables `DD_AGENT_HOST` and `DD_DOGSTATSD_PORT`.
+By default, runtime metrics from your application are sent every 10 seconds to the Datadog Agent with DogStatsD. Make sure that [DogStatsD is enabled for the Agent][2]. If your Datadog Agent DogStatsD address differs from the default `localhost:8125`, use the `WithDogstatsdAddress` option or the environment variables `DD_AGENT_HOST` and `DD_DOGSTATSD_PORT`. See the `WithDogstatsdAddress` documentation for information on how to configure the connection to [DogStatsD][3]
 
-If you are running the Agent as a container, ensure that `DD_DOGSTATSD_NON_LOCAL_TRAFFIC` [is set to true][3], and that port `8125` is open on the Agent. Additionally, for:
+If you are running the Agent as a container, ensure that `DD_DOGSTATSD_NON_LOCAL_TRAFFIC` [is set to true][4], and that port `8125` is open on the Agent. Additionally, for:
 
-- **Kubernetes**: You _must_ [bind the DogstatsD port to a host port][4].
-- **ECS**: [Set the appropriate flags in your task definition][5].
+- **Kubernetes**: You _must_ [bind the DogstatsD port to a host port][5].
+- **ECS**: [Set the appropriate flags in your task definition][6].
 
 ## Data Collected
 
 The following metrics are collected by default after enabling Go metrics.
 
 {{< get-metrics-from-git "go" >}}
```

Comment:
Question unrelated to this change: is this being rendered correctly at https://github.com/DataDog/documentation/blob/master/content/en/tracing/runtime_metrics/go.md? Lines 41 and 47 look like they're supposed to be linking somewhere, but don't.

Reply:
These get rendered into HTML where that shows up. You should be able to see the page once the build finishes here: https://docs-staging.datadoghq.com/knusbaum/update-dogstatsd-doc/tracing/runtime_metrics/go